agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/ConfigOverride.java

```diff
@@ -71,19 +71,22 @@ static Config getConfig(Configuration config) {
     if (!config.instrumentation.springScheduling.enabled) {
       properties.put("otel.instrumentation.spring-scheduling.enabled", "false");
     }
-    if (!config.preview.instrumentation.apacheCamel.enabled) {
-      properties.put("otel.instrumentation.apache-camel.enabled", "false");
+    if (config.preview.instrumentation.apacheCamel.enabled) {
+      // apache-camel instrumentation is off by default
+      properties.put("otel.instrumentation.apache-camel.enabled", "true");
```

Comment:
this won't work, `apache-camel` is enabled by default in otel

Reply:
that's the confusing part.. preview instrumentation is off by default. are you saying that otel will enable it by by disregarding the json config setting?

Reply:
check out https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/suppressing-instrumentation.md

Reply:
i only found apache camel and grizzly in that list.. quartz and springIntegration are not mentioned though.